### DISCUSSION TITLE
Site/Global Assets to the PersonaBar

### DISCUSSION THREAD

**Discussion by alexsinclersantos** [dnnsoftware/Dnn.Platform#4663] [Ideas]
Not sure if this has been answered already, but do we have an ETA when Site/Global Assets are going to the PersonaBar? I just installed DNN after 2 years of not using it at all and I was really hoping this was addressed already. It seems like that's the only option that still relies on pages to show the module. Thank you everyone.

**Comment by mitchelsellers** [comment]
We will accept community contributions to help get us to this point, however, this feature, like others will be implemented when we receive a contribution.
